kickstarts/partials/post/ui_compile.ks.erb

```diff
@@ -9,6 +9,7 @@ popd
 # Service UI
 pushd /opt/manageiq/manageiq-ui-service
   yarn install --production
```

Comment:
You should probably remove `--production` here.

`glob` is a devDependency,
and as of https://github.com/ManageIQ/manageiq-ui-service/pull/1576, all build dependencies will be `devDependencies` (as opposed to runtime dependencies which will all be in `dependencies`)